src/parse/archie/index.js

```diff
@@ -2,10 +2,15 @@ import validateArgs from 'aproba';
 import archieml from 'archieml';
 import docsToArchie from './_docsToArchie';
 
-export default async function parseArchie(docId) {
+export default async function parseArchie(
+  docId,
+  { suggestionsViewMode = 'PREVIEW_WITHOUT_SUGGESTIONS' } = {},
```

Comment:
Let's keep the default out of this, just let the default fall down to docs.get. That way it's clear where the default lives.

```suggestion
  { suggestionsViewMode } = {},
```